src-electron/server/ws-server.js

```diff
@@ -134,9 +144,10 @@ function sendWebSocketData(socket, category, payload) {
     category: category,
     payload: payload,
   }
-  doSend(socket, obj)
+  return new Promise((resolve, reject) => {
+    resolve(doSend(socket, obj))
```

Comment:
why is doSend wrapped within a Promise block? doSend() does not intentionally return any value to be consumed later.

Reply:
duplicate comment

Reply:
We discussed adding this to make sure something was returned, ill just take it out for now because it is not being used